ecc_enclave/enclave/enclave.cpp

```diff
@@ -22,11 +22,6 @@ extern sgx_ec256_public_t enclave_pk;
 // this is tlcc binding
 sgx_ec256_public_t tlcc_pk = {0};
 
-// TODO: replace with secure key generated in `common/enclave/cc_data.cpp` when we also create
-// cc-encryption/decryption key-pair .. state encryption key; hardcoded for debugging
-sgx_aes_gcm_128bit_key_t state_encryption_key = {
```

Comment:
I guess you squashed a bit too much. Luckily it does not conflict!

Reply:
Arghh, good catch. turns out i screwed up in rebase and merged my changes into your commit (and also forgot to add some changes i had locally).
=> don't merge yet but wait for force-push below (and corresponding successful travis build)